scalecube-fixtures-tck-flow/src/test/java/io/scalecube/test/fixtures/FailingTest.java

```diff
@@ -0,0 +1,23 @@
+package io.scalecube.test.fixtures;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+// @WithFixture(value = FailingFixtureConstructor.class)
```

Comment:
so does it work (fail in the fixture constructor)?

Reply:
Check out, disable the comment and get this:

> [INFO] Running io.scalecube.test.fixtures.FailingTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.004 s <<< FAILURE! - in io.scalecube.test.fixtures.FailingTest
[ERROR] failureDetection{TestInfo}  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException